docs/reference/commandline/swarm_init.md

```diff
@@ -36,6 +36,7 @@ Options:
       --max-snapshots uint              Number of additional Raft snapshots to retain
       --snapshot-interval uint          Number of log entries between Raft snapshots (default 10000)
       --task-history-limit int          Task history retention limit (default 5)
+      --data-path-port uint             Data path port number
```

Comment:
Could you move this up; below `--data-path-addr`? The cli sorts this alphabetically in the output.

This will also have to be updated to match the actual output (if you make my other suggested changes for the flag description)